test/helpers/http-server.ts

```diff
@@ -0,0 +1,157 @@
+import http = require('http');
+import https = require('https');
+import net = require('net');
+import express = require('express');
+import pify = require('pify');
+import pem = require('pem');
+import bodyParser = require('body-parser');
+import tempy = require('tempy');
+
+export type BaseServerOptions = {
+	protocol: 'HTTP' | 'HTTPS' | 'socket';
```

Comment:
I think these should be lowercase. They are used as identifiers, so it's ok that they don't match the actual acronym.